Stats Json-related updates

almossawi opened the pull request:
That a country’s entire data set exists in the json is not an invariant. This update sets missing attributes to 0. I have also updated the code that generated the set of arbitrary countries for chart 3 to fix a bug that could potentially cause the entire set not to be populated.

The other change is the use of data-static-url rather than a hard-coded url.

almossawi committed:
Use data-static-url to get json's url

almossawi committed:
Override the url for local development

pmclanahan commented on mrburns/main/static/js/main.js:
```diff
@@ -43,30 +43,29 @@ Date.prototype.addHours= function(h){
+        
     //get data file from 2 mins ago
-    //TODO revisit once in dev
+    url = 'https://webwewant.allizom.org/static/'; //TODO for local development only
```
For local development just put `STATIC_URL = 'https://webwewant.allizom.org/static/'` in your `mrburns/settings/local.py` file. That will cause all static files to be loaded from there though. If you want a separate setting for data file specifically I can make that happen.

almossawi replied:
I need it only for the data file, if possible.

almossawi committed:
Stats country lookup in json update

That a country’s entire data set exists in the json is not an
invariant. This update sets missing attributes to 0. I have also
updated the code that generated the set of arbitrary countries for
chart 3 to fix a bug that could potentially cause the entire set not to
be populated.
